week-4/F-object-keys/exercise-part-2.js

```diff
@@ -0,0 +1,46 @@
+/*
+
+Objects very often contain other objects. You'll encounter nested objects all the time in the wild. This exercise will give you practice in getting the keys of objects inside other objects.
```

Comment:
People might not know phrases like "in the wild" so that might just be extra noise when trying to understand the requirements - the first sentence already says that they are common. Maybe just reiterate that objects within objects are called "nested" so that people get that again when they come to read the exercise?